doc/lua_api.md

```diff
@@ -1099,6 +1099,11 @@ Table used to specify how a sound is played:
 
     -- Available since feature `sound_params_start_time`.
 
+    keep_time = 0.0,
+    -- Define time window which allow sound to be sent to players
```

Comment:
I don't have a good suggestion but it feels like we're trading one hack¹ for another by adding this parameter.
Maybe the server should just track all sounds for a duration of 10 minutes and automatically re-send them? It would be up to the client to skip sounds where `start_time >= sound_length`

¹: `max_hear_distance` is the previous hack

Reply:
I used 600s as the default `keep_time` value in the last version.